biz.aQute.bndlib/src/aQute/bnd/osgi/repository/BridgeRepository.java

```diff
@@ -263,6 +289,17 @@ public Resource get(String bsn, Version version) throws Exception {
 		return resourceInfo.resource;
 	}
 
+	public ResourceInfo getInfo(String bsn) throws Exception {
+		Map<Version, ResourceInfo> map = index.get(bsn);
+		if (map == null)
+			return null;
+
+		return MapStream.of(map.entrySet())
```

Comment:
Since you only want the values, just do `map.values().stream()`

Reply:
:|